src/main/java/mods/railcraft/common/util/misc/EntityIDs.java

```diff
@@ -35,6 +35,7 @@
     public static final int CART_LOCO_STEAM_SOLID = 30;
     public static final int CART_LOCO_STEAM_MAGIC = 31;
     public static final int CART_LOCO_ELECTRIC = 32;
+    public static final int CART_COMMAND_BLOCK = 50;
```

Comment:
maybe it this could be converted to an enum to prevent conflicts like this in the future?